himin/day3/두 개 뽑아서 더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(numbers) {
+    const sum = new Set();
+  
+    for (let i = 0; i < numbers.length; i++) {
+        for (let j = i + 1; j < numbers.length ; j++) {
+         sum.add(numbers[i] + numbers[j]);   
+        }
+    }
+      
+      return [...sum].sort( (a, b) => a - b) ;
+  }
```

Comment:
```suggestion
function solution(numbers) {
    const sum = new Set();
  
    for (let i = 0; i < numbers.length; i++) {
        for (let j = i + 1; j < numbers.length ; j++) {
         sum.add(numbers[i] + numbers[j]);   
        }
    }
      
    return [...sum].sort( (a, b) => a - b) ;
}
```
set은 자료구조입니다. MDN에서는 아래와 같이 정의해주네요. 
```
Set 객체는 [원시값](https://developer.mozilla.org/ko/docs/Glossary/Primitive)이나 객체 참조 값 등 모든 유형의 고유 값을 저장할 때 사용할 수 있습니다.
```
그런 의미에서 sum이라는 단어의 의미는 `합/총합/총계`인데 이 Set이라는 자료구조에 적합한 네이밍인지 궁금합니다 ㅎㅎ